README.md

```diff
@@ -176,6 +176,34 @@ $ helm dt unwrap kibana-10.4.8.wrap.tgz demo.goharbor.io/helm-plugin/ --yes
 
 If your wrap includes bundled artifacts (if you wrapped it using the `--fetch-artifacts` flag), they will be also pushed to the remote registry.
 
+Unwrapping a Helm chart to a different target than the original using `--push-repository` flag.
+By unwrapping the Helm chart to a target OCI registry the `dt` tool will unwrap the wrapped file, proceed to push the container
+images into the registry you have specified by `--push-repository`, relocate the references from the Helm chart and preserve the original target
```

Comment:
Silly comment. For consistency it would be great if you can remove this line breaks on any single paragraph.